components/debug/memory-viewer.tsx

```
@@ -0,0 +1,413 @@
+            <div className="flex border-b">
+                {subTabs.map((tab) => (
+                    <button
+                        key={tab.id}
+                        onClick={() => setActiveSubTab(tab.id)}
+                        className={cn(
+                            "flex flex-col items-start gap-1 px-4 py-3 text-sm border-b-2 transition-colors",
+                            activeSubTab === tab.id
+                                ? "border-primary text-primary"
+                                : "border-transparent text-muted-foreground hover:text-foreground hover:border-muted-foreground"
+                        )}
+                    >
```

Biome (1.9.4)

[error] Provide an explicit type prop for the button element.

The default type of a button is submit, which causes the submission of a form when placed inside a `form` element. This is likely not the behaviour that you want inside a React application.
Allowed button types are: submit, button or reset

(lint/a11y/useButtonType)